src/org/freedesktop/gstreamer/glib/NativeObject.java

```diff
@@ -133,7 +133,7 @@ public int hashCode() {
      * After calling this method this object should not be used.
      */
     public void invalidate() {
-        LOG.log(LIFECYCLE, () -> "Invalidating object " + this + " = " + getRawPointer());
+        LOG.log(LIFECYCLE, () -> "Invalidating object " + getClass().getName() + " = " + getRawPointer());
```

Comment:
Reason?  Probably makes sense.

Reply:
Some toString() method calls native methods and cost a lot of time. I replace by the same as in NativeObject::dispose()

Reply:
Thanks.  Good reason and good to know.  Interestingly, looking back at the old bindings, at least I changed this to be lazy - it was called even when not logging - https://github.com/gstreamer-java/gstreamer-java/blob/master/gstreamer-java/src/org/gstreamer/lowlevel/NativeObject.java#L113